.github/workflows/ansible-test.yml

```diff
@@ -6,6 +6,19 @@
 
 name: CI
 on:
+  # Manual triggers
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: false
+        default: 'warning'
+        type: choice
+        options:
+        - info
+        - warning
+        - debug
```

Comment:
👀 where is this used?

Reply:
Just a quick C&P, wanted to get the option available after the gh runners went down earlier. Might use it later though

Reply:
Oh I see, `workflow_dispatch` doesn't need any inputs to be defined, if you just want a trigger with no params